src/assets/styles/layout.scss

```diff
@@ -37,3 +37,7 @@ form {
   color: color($colors, primary, base);
   -webkit-text-stroke: .4px color($colors, primary, base);
 }
+
+.alert-md.select-alert div[ng-reflect-ng-switch] {
```

Comment:
`.alert-md`? this will only change on Android.

Reply:
It's not specific to Android? alert-md is an Angular class I believe. You can see it in chrome:

![image](https://user-images.githubusercontent.com/8069294/45293351-da73b680-b4ef-11e8-9c2c-1df59d0782f7.png)

Reply:
I'll change it to `ion-alert` - i tried that first but it didn't work (which is why i used `alert-md`). Just tried again and it is so bare with

Reply:
Updated 👌